youths/tests/test_youth_profiles_graphql_api.py

```diff
@@ -0,0 +1,314 @@
+from collections import OrderedDict
+from string import Template
+
+from youths.tests.factories import ProfileFactory, YouthProfileFactory
+
+
+def test_anon_user_query_should_fail(rf, youth_profile, anon_user_gql_client):
+    request = rf.post("/graphql")
+    request.user = anon_user_gql_client.user
+
+    query = """
+        {
+            youthProfile(profileId: 1) {
+                ssn
+                gender
+                schoolClass
+            }
+        }
+    """
+    expected_data = OrderedDict([("youthProfile", None)])
```

Comment:
you can avoid using OrderedDict and make tests slightly more readable by casting the result of `gql_client.execute()` to a `dict`:
```
expected_data = {"youthProfile": None}
executed = anon_user_gql_client.execute(query, context_value=request)
assert dict(executed["data"]) == expected_data
```

Reply:
Thank you! I'll change them